packages/components/card/README.md

```diff
@@ -20,11 +20,11 @@ npm --save install @commercetools-uikit/card
 Additionally install the peer dependencies (if not present)
 
 ```
-yarn add react
```

Comment:
This shouldn't be part of this PR.
After refreshing this branch with `main` we should run the readmes generator again.